pilot/pkg/features/pilot.go

```diff
@@ -761,6 +761,12 @@ var (
 	// This is a feature flag, can be removed if protobuf proves universally better.
 	KubernetesClientContentType = env.Register("ISTIO_KUBE_CLIENT_CONTENT_TYPE", "protobuf",
 		"The content type to use for Kubernetes clients. Defaults to protobuf. Valid options: [protobuf, json]").Get()
+
+	// This is an experimental feature flag, can be removed once it became stable, and should introduced to Telemetry API.
+	MetricRotationInterval = env.Register("METRIC_ROTATION_INTERVAL", 0*time.Second,
```

Comment:
Since this is a new feature and we are nearing the release of 1.18, could you please provide more details about why this is necessary for the 1.18 release? This will help other RMs make decisions. 

Personally, I have no objections, especially since this feature is disabled by default.

Reply:
Or I think the releasenotes should be a little bit more detailed, specifically mentioning that this is an experimental feature.

Reply:
Yes, as you said, this's an experimental feature and disabeld by default, hope to get more feedback from end-user, that's why I want to land it in v1.18(usually istio donot introduce new feature in minor release).